iree/tools/opt_main.cc

```diff
@@ -80,6 +80,7 @@ int main(int argc, char **argv) {
   mlir::iree_compiler::registerHALTargetBackends();
   mlir::iree_compiler::registerLinalgToSPIRVPasses();
   mlir::iree_compiler::registerHLOToLinalgPasses();
+  mlir::iree_compiler::registerLinalgToLLVMPasses();
```

Comment:
Could add this line to `iree/samples/custom_modules/dialect/custom_opt.cc` as well? :)

Reply:
Further, the question arises if this and at least the two pass registrations above should be moved to `registerAllIreePasses()`.

Reply:
I think it should be ok to move codegen passes registration to `registerAllIreePasses()`. 
@GMNGeoffrey thoughts ?

Reply:
@GMNGeoffrey Gentle ping :)

Reply:
Whoops missed this. So the question is whether registering iree passes should include these. I mean given the name `AllIreePasses` I would expect it to include all of them :-D What's the reason for the distinction now? Do we have scenarios where we want to register the rest but not these ones? It's good if things are factored such that you can get only the passes for some specific IREE backend, but it still seems reasonable that we have a function that just does all of them, even if that one has separate calls to split out by backend.